### ISSUE TITLE
Project Listing: Backend Implementation

### ISSUE THREAD

**Issue by BluuArc** [lugatuic/lug.cs.uic.edu-2018#23]
Implement API for requesting projects

**Comment by BluuArc** [comment]
From mock data
```
[
  {
    name: 'Project Name',
    status: 'incomplete',
    description: 'project description',
    smallImage: '',
    largeImage: '',
    wikiLink: 'link to wiki for this project',
    githubLink: 'link to github for this project',
    startDate: new Date(),
    projectLead: 'Some name'
  },
]
````

Possible fields for status: active, inactive, complete

`GET /api/projects` - returns an array of projects

**Comment by BluuArc** [comment]
Implemented by #61 , merged with #62
